src/main/java/com/artipie/api/artifactory/AddUpdateUserSlice.java

```diff
@@ -74,8 +75,9 @@ public Response response(final String line, final Iterable<Map.Entry<String, Str
                         passw -> passw.map(
                             haspassw -> this.settings.credentials()
                                 .thenCompose(
-                                    cred -> cred.add(username, haspassw)
-                                        .thenApply(ok -> new RsWithStatus(RsStatus.OK)))
+                                    cred -> cred.add(
+                                        username, haspassw, Credentials.PasswordFormat.PLAIN
```

Comment:
@olenagerasimova I see that passwords were stored were stored in plain format before these changes, but maybe we should stick to sha256 format by default?